src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -166,45 +166,44 @@ private bool BuildPackage(PackageBuilder builder, string outputPath = null, bool
                 PrintVerbose(outputPath, builder);
             }
 
-            using (var package = new PackageArchiveReader(outputPath))
-            {
+            using var package = new PackageArchiveReader(outputPath);
```

Comment:
This seems like it's only changing from old usage to new usage. Is there an analyer fix here?

Reply:
I followed the example in this [doc](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations).

[IDE0063](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0063?source=docs) is the analyzer to simplify `using` statements in `C# language version >= 8`. This analyzer has a fixer whose light bulb suggestions can be observed in the image below.

![image](https://user-images.githubusercontent.com/52756182/103948833-4762df80-50ef-11eb-81c5-d83bd0fde781.png)

Reply:
Given that it's a stylistic fix, rather than a functional one, I'd normally say, avoid doing that to not introduce unnecessary changes. Especially when readability is not necessarily changing too much. 

Given all that, I won't block on that, you can make a judgment call/discuss this feedback with others from the team.

Reply:
> Given that it's a stylistic fix, rather than a functional one, I'd normally say, avoid doing that to not introduce unnecessary changes. Especially when readability is not necessarily changing too much.

Sure, I agree. I also feel setting `Language Version` to 8 to `core` libraries a bit restrictive because `net5.0` already defaults to `Language Version 9`.  Since all of our projects targets multiple TFMs including `net472`, I think we will be struck to `C# 7.3` which is the default language version for all `.NET framework` projects.

I prefer reverting my last commit to this branch where we added '<LangVersion>8</LangVersion>` for few projects unless other team members has additional feedback.

Reply:
I'm not suggesting going back to 7.3 actually. 
I'm merely suggesting address only CA2000 warnings in this PR. 

We should move to 8 wherever we can.

Reply:
Got it. Thanks, Nikolche for clarification. AFAIK, I modified `using` statement while fixing CA2000 warnings in this PR but not in other places. I will fix the nit pics suggested in other comments in the next commit.